pkg/controller/elasticsearch/sset/fixtures.go

```diff
@@ -23,6 +24,27 @@ type TestSset struct {
 	Status      appsv1.StatefulSetStatus
 }
 
+func (t TestSset) Pods() []runtime.Object {
+	podNames := PodNames(t.Build())
+	pods := make([]runtime.Object, t.Replicas)
+	for i, podName := range podNames {
+		pod := &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      podName,
+				Namespace: t.Namespace,
+				Labels: map[string]string{
+					label.VersionLabelName:     t.Version,
+					label.ClusterNameLabelName: t.ClusterName,
+				},
+			},
+		}
+		label.NodeTypesMasterLabelName.Set(t.Master, pod.Labels)
+		label.NodeTypesDataLabelName.Set(t.Data, pod.Labels)
+		pods[i] = pod
+	}
```

Comment:
Could this not reuse `TestPod`?

Reply:
`TestPod` seems suitable when creating Pods "from scratch"
In my case I can generate them automatically from the `TestSset` without duplicating all the needed information.